### ISSUE TITLE
The Azure Providers team at HashiCorp is hiring!

### ISSUE THREAD

**Issue by katbyte** [hashicorp/terraform-provider-azurerm#26754]
The Azure Providers team here at HashiCorp is now hiring! 

We are looking for a full time engineers/maintainer to join out team - if you're interested in working at HashiCorp to maintain this project you can find more information about the role and how to apply on the regional job postings below:

Senior Engineer:
- [Canada](https://www.hashicorp.com/career/6330236)
- [US](https://www.hashicorp.com/career/6340203)

Engineer:
- [Canada](https://www.hashicorp.com/career/6127507)
- [US](https://www.hashicorp.com/career/6127502)

If you have any questions about the role please don't hesitate to contact via [email](mailto:kt@katbyte.me) or in our community slack

**Comment by aristosvo** [comment]
I couldn't log into Greehouse, the roles are also on the website:
- [US](https://www.hashicorp.com/career/6127502)
- [Canada](https://www.hashicorp.com/career/6127507)

**Comment by katbyte** [comment]
Thanks @aristosvo - updated my comment with the non greenhouse links

**Comment by ArcTheMaster** [comment]
@katbyte is this ticket still open ? Can I pm you regarding the topic :)

**Comment by katbyte** [comment]
@ArcTheMaster yes, as long as this is still open we are still hiring 🙂 Just updated with the new senior role opening

please reach out on the community slack
